zadaca5/src/zadatak3/Complex.java

```diff
@@ -0,0 +1,48 @@
+package zadatak3;
+
+public class Complex {
+    private int rPart;
+    private int iPart;
+    public Complex () {
+        this(0,0);
+    }
+    public Complex (int rPart, int iPart) {
+        this.rPart = rPart;
+        this.iPart = iPart;
+    }
+    public int Re(){
+        return this.rPart;
+    }
+    public int Im(){
+        return this.iPart;
+    }
+    public Complex add(Complex z1){
+        int z1Re = z1.Re();
+        int z1Im = z1.Im();
+        Complex sumComplex = new Complex(this.rPart+z1Re, this.iPart+z1Im);
+        return sumComplex;
```

Comment:
Like that you used attributes directly of the calling objects and leveraged `Im()` and `Re()` for the input parameter, this creates a good distinction between the two.

No need for additional variables just: `return new Complex(this.rPart+z1.Re(), this.iPart+z1.Im())`

Also the parameter could have been called simply `z`,

Reply:
Noted, will adjust :)